actor-core/src/main/java/org/sunbird/actor/core/BaseActor.java

```diff
@@ -29,7 +30,8 @@ public void onReceive(Object message) throws Throwable {
     if (message instanceof Request) {
       Request request = (Request) message;
       String callerName = request.getOperation();
-      ProjectLogger.log("BaseActor onReceive called for operation : " + callerName);
+      ProjectLogger.log(
+          "BaseActor onReceive called for operation : " + callerName, LoggerEnum.INFO);
```

Comment:
Change ...

From:
"BaseActor onReceive called for operation : "

To:
"BaseActor: onReceive called for operation: "